src/System.Net.HttpListener/src/System/Net/Managed/ChunkedInputStream.cs

```diff
@@ -76,7 +76,7 @@ public override int Read(byte[] buffer, int offset, int count)
             return EndRead(ares);
         }
 
-        public override IAsyncResult BeginRead(byte[] buffer, int offset, int count, AsyncCallback cback, object state)
+        public override IAsyncResult BeginRead(byte[] buffer, int offset, int size, AsyncCallback cback, object state)
```

Comment:
Why did you change this from count to size?  The base implementation uses "count"... I don't see a good reason to move away from that.